luxtronik/discover.py

```diff
@@ -32,35 +34,37 @@ def discover() -> list[(str, int)]:
 
         # send AIT magic broadcast packet
         server.sendto(LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode(), ("<broadcast>", port))
-        LOGGER.debug("Sending broadcast request %s", LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode())
+        LOGGER.debug(
+            "Sending broadcast request %s", LUXTRONIK_DISCOVERY_MAGIC_PACKET.encode()
+        )
 
         while True:
             try:
-                res, con = server.recvfrom(1024)
-                res = res.decode("ascii", errors="ignore")
```

Comment:
mypy: error: Incompatible types in assignment (expression has type "str", variable has type "bytes")  [assignment]
Double use of var res!